exercise-book/src/nrf52-tools.md

```diff
@@ -125,9 +121,9 @@ cargo install cargo-binutils
 rustup +stable component add llvm-tools
 ```
 
-### General purpose tools
+### Third-party tools written in Rust
```

Comment:
Nit: `flip-link` is first party, as it is maintained by us... This should be worked out a bit better.

Reply:
#116